### ISSUE TITLE
crossencoder with one sentence input for text classification

### ISSUE THREAD

**Issue by ironllamagirl** [UKPLab/sentence-transformers#1272]
I have a text classification problem where I need to classify text into one of 4 categories. I would like to use sbert but read that crossencoder only takes pair input.

How do I go about doing this? is there an example code? Thank you so much

**Comment by timpal0l** [comment]
Since this is a classification problem rather than a semantic search problem, you could just use a pretrained bert model as a bi-encoder.

**Comment by nreimers** [comment]
You can use the CrossEncoder also for individual texts. So you pass a list with individual sentences instead of sentence pairs

**Comment by ironllamagirl** [comment]
@nreimers  Oh I did not know that. I will try it out
Thanks!!

**Comment by ddofer** [comment]
I'll note that the examples and documentation are not clear that one can use just a single input sentence. 
(Also, i'm getting a tokenizer/padding error when using the CE + softmax evaluator, but that's a seperate issue)
